extensions/adobe/experience/adcloud/campaigndetails.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/campaigndetails",
+  "title": "Ad Campaign Detail",
+  "type": "object",
+  "description": "Adobe AdCloud Ad Campaign Hierarchy Details",
+  "definitions": {
+    "campaignDetails": {
+      "properties": {
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/accountId": {
+          "title": "Account ID",
+          "type": "integer",
+          "description":
+          "Unique identifier for the account this event is associated with."
+        },
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/campaignId": {
+          "title": "Campaign ID",
+          "type": "integer",
+          "description": "Unique identifier for the campaign which this event is associated with."
+        },
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/placementId": {
+          "title": "Placement ID",
+          "type": "integer",
+          "description": "Unique identifier for the placement which this event is associated with."
+        },
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/packageId": {
+          "title": "Package ID",
+          "type": "integer",
+          "description": "Unique identifier for the package which this event is associated with."
+        },
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/advertiserId": {
+          "title": "Advertiser ID",
+          "type": "integer",
+          "description": "Unique identifier for the advertiser which this event is associated with."
+        },
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/experimentId": {
+          "title": "Experiment ID",
+          "type": "integer",
+          "description": "Unique identifier for the experiment which this event is associated with."
+        },
+        "shttps://ns.adobe.com/experience/adcloud/campaigndetails/ampleGroupId": {
+          "title": "Sample Group ID",
+          "type": "integer",
+          "description": "Unique identifier for the sample group which this event is associated with."
+        },
+        "https://ns.adobe.com/experience/adcloud/campaigndetails/currency": {
+          "title": "Account Currency",
+          "type": "string",
+          "examples": ["USD", "EUR"],
+          "pattern": "^[A-Z]{3}$",
+          "description": "The ISO 4217 billing currency code for the AdCloud account this event is associated with."
```

Comment:
Should we be specifically referring to "AdCloud" here?

Reply:
Removed AdCloud.